packages/phpstan-rules/tests/Rules/CheckConstantExpressionDefinedInConstructOrSetupRule/Fixture/InsideSetup.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\PHPStanRules\Tests\Rules\CheckParentChildMethodParameterTypeCompatibleRule\Fixture;
+
+class InsideSetup
+{
+    private const A = 'a';
+
+    public function setUp()
+    {
+        $a = self::A;
+    }
```

Comment:
What about:

- __DIR__: . '/static.txt'; - error
- __DIR__ . $this->getValues(); - skip

Reply:
implemented.